specs/src/specs/data_structures.md

```diff
@@ -495,7 +495,8 @@ For shares **with a reserved namespace ID through [`NAMESPACE_ID_MAX_RESERVED`](
 For shares **with a namespace ID above [`NAMESPACE_ID_MAX_RESERVED`](./consensus.md#constants) but below [`PARITY_SHARE_NAMESPACE_ID`](./consensus.md#constants)**:
 
 - The first [`NAMESPACE_ID_BYTES`](./consensus.md#constants) of a share's raw data `rawData` is the namespace ID of that share, `namespaceID`.
-- The remaining [`SHARE_SIZE`](./consensus.md#constants)`-`[`NAMESPACE_ID_BYTES`](./consensus.md#constants) bytes are request data. In other words, the remaining bytes have no special meaning and are simply used to store data.
+- If this is the first share of a message, the next 1-10 bytes contain a [varint](https://developers.google.com/protocol-buffers/docs/encoding) of the uint64 length of the message that follows. If this isn't the first share of a message, the next 1-10 bytes are used for request data.
```

Comment:
I think "request" is the wrong term here. @adlerjohn adapted it from the tendermint paper (which followed canonical BFT consensus research nomenclature). In our context, "request" is too vague or even confusing.

Reply:
Agreed. Will fix in https://github.com/celestiaorg/celestia-app/pull/856#discussion_r993628147 after we align on better names

Reply:
Following up here, fixed in https://github.com/celestiaorg/celestia-app/pull/856/commits/1d1ec47454f28526bdcfc44db95636f39e639fe7